### ISSUE TITLE
Charts loading mechanism when using pagination

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/visualizer#689]
### Description:
This was discussed before, and it was indeed an edge case. When you refresh the page sooner than 2 seconds, the charts are not loading.

But when using pagination, the same thing happens in some cases, and I think we can consider finding a solution, as indeed refreshing the page really fast is not an actual use case, but navigating throw the pages is.

### How to reproduce:
1. Add a custom number of charts per page
2. Navigate through page 1, page 2, next, previous etc

### Expected behavior:
The charts to load

### Current behavior:
In some cases, the charts don't load

### Reference:
https://www.loom.com/share/b719186e13aa4172a7eb61e0a173ab12

### Technical info
* WordPress version: latest
* Plugin version: 3.4.3

**Comment by DanyCaissy** [comment]
I agree this bug can be cumbersome and it often happens to me during normal usage. We should try to resolve it as it can hinder the user experience.
